src/lib/_imports/elements/table--basic.css

```diff
@@ -127,7 +119,106 @@ table p,
 :--s-paragraph-table p {
   margin-top: 0;
 }
+
 table p:last-child,
 :--s-paragraph-table p {
   margin-bottom: 0;
 }
+
+/* SEE: https://css-tricks.com/responsive-data-tables/ */
+@media (max-width: 767px) {
+  /* overwrite Core CMS site.css */
+  table,
+  :--s-paragraph-table {
+    --font-size: var(--global-font-size--medium, 14px);
+    --cell-pad-vert: 1.2em;
+    --cell-pad-horz: 2.2em;
+
+    font-family: var(--global-font-family--sans--portal);
+    font-size: var(--font-size);
+
+    border-top: var(--global-border-width--normal) solid var(--global-color-primary--dark);
+    border-bottom: var(--global-border-width--normal) solid var(--global-color-primary--dark);
+  }
+
+
+  :is(table, thead, tbody, th, td, tr) {
+    display: block;
+  }
+
+  tr > :is(td, th):not(:last-of-type),
+  :--s-paragraph-table p:not(:last-of-type) {
+    border-bottom-width: var(--global-border-width--normal);
+    border-bottom-style: solid;
+    border-bottom-color: var(--global-color-primary--light);
+  }
+
+  tr:not(:last-of-type) {
+    border-bottom: var(--global-border-width--normal) solid var(--global-color-primary--light);
+  }
+
+  /* To make cells look like left-aligned labels */
+  td:before {
+    position: absolute;
+    width: 45%;
+    padding-right: 10px;
+    white-space: nowrap;
+    font-weight: bold;
+    overflow: hidden;
+    text-overflow: ellipsis;
+
+    /* To mimic cell padding */
+    left: var(--cell-pad-horz);
+  }
+
+  tbody:not(thead + tbody) > tr:first-child > :is(td, th) {
+    border-top: none;
+  }
+
+  /* To remove border (that should probably be in Core anyway) */
+  table.has-data-cols {
+    /* To hide table headers (but not `display: none;`, for a11y) */
+    & thead tr {
+      position: absolute;
+      top: -9999px;
+      left: -9999px;
+    }
+
+    /* To make each cell behave like a "row" */
+    & td {
+      position: relative;
+      padding-left: 50%;
+    }
+
+    /* To label the cells */
+    /* RFE: Add `data-label` to each cell so we can use `attr(data-label)` */
+    & td:before {
+      content: attr(data-col);
+    }
+
+    & td:not(:first-child) th:first-child,
+    td:not(:first-child) td:first-child {
+      padding-inline-start: 50%;
+    }
+
+    & td:not(:last-child) th:last-child,
+    td:not(:last-child) td:last-child {
+      padding-inline-end: 50%;
+    }
+
+    & .c-pill {
+      min-width: unset;
+      padding-inline: unset;
```

Comment:
Removing the `min-width` is great. It should have some padding, at most the same. Unless design confirmed otherwise.

```suggestion
```